lib/index.js

```diff
@@ -221,6 +220,29 @@ module.exports.render = function(options) {
     };
   }
 
+  if (options.functions) {
+    Object.keys(options.functions).forEach(function(signature) {
+      var callback = options.functions[signature];
+
+      options.functions[signature] = function() {
+        var args = Array.prototype.slice.call(arguments),
+            bridge = args.pop();
+        // var args = Array.slice(arguments);
+
+        function done(data) {
+          bridge.success(data);
+        }
+
+        var result = callback.apply(this, args.concat(done));
+
+        if (result) {
+          done(result);
+        }
+      };
+    });
+  }
```

Comment:
If possible, we need to reuse the same logic for CLI usage. Just note for self. Let me know if / whenever you are adding the CLI support (before that). We need to adapt and enhance on the way jscs, jshint and other linters support custom importers. I will then explain it. :)